app/services/teacher_training_api/check_courses_for_missing_providers.rb

```diff
@@ -30,11 +30,11 @@ def courses_with_missing_providers
     end
 
     def message
-      return "There are no courses with missing providers for recruitment cycle year #{recruitment_cycle_year}." unless courses_with_missing_providers.count.positive?
+      return "[#{Rails.env}] [#{Time.zone.now.to_fs(:govuk_date_and_time)}] CheckCoursesForMissingProviders - There are no courses with missing providers for recruitment cycle year #{recruitment_cycle_year}." unless courses_with_missing_providers.count.positive?
```

Comment:
Can we make this consistent with the Provider message checker format? Something like this:
```suggestion
      return "[#{Rails.env}] Course Provider Checker Results #{Time.zone.now.to_fs(:govuk_date_and_time)} for #{recruitment_cycle_year}:\nThere are no courses with missing providers for recruitment cycle year." unless courses_with_missing_providers.count.positive?
```